src/java/org/apache/cassandra/repair/LocalSyncTask.java

```diff
@@ -174,6 +175,9 @@ public synchronized void onFailure(Throwable t)
         if (active)
         {
             active = false;
+            String message = String.format("Sync failed using session %s between %s and %s on %s: %s", 
+                                           desc.sessionId, nodePair().coordinator, nodePair().peer, desc.columnFamily, t.getMessage());
```

Comment:
This line is long and doesn't fit screen when reviewed in GH, i.e., difficult to review.